pkgs/development/tools/revive/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildGoModule, fetchFromGitHub, lib, bash }:
+
+buildGoModule rec {
+  pname = "revive";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "mgechev";
+    repo = "revive";
+    rev = "v${version}";
+    sha256 = "sha256-V0YWp4zQduW+cnuiVoEAPxDnsA4OR0TnJdoOF9TdRjw=";
+  };
+
+  doCheck = false;
```

Comment:
@maaslalani the comment you added doesn't really explain anything

Reply:
Sorry about that, is the current comment more descriptive / acceptable?

Reply:
Yeah that's better although typically I add a comment that explains what the specific issue is rather than which tests are broken e.g:

"several tests use networking" etc

I've not ran the tests yet but looking at the code I'm a bit confused why there are issues :thinking: